File: 기음#/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cs
using System;
using System.Linq;

public class Solution {
    public int solution(int[] num_list) {
        int a = 1;
        for (int i=0;i<num_list.Length;i++){
            a*=num_list[i];
        }
        if (a<Math.Pow(num_list.Sum(),2)){
            return 1;
        }
        return 0;
    }
}

File: C++/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int a = 1;
    int b = 0;
    for (int i=0;i<num_list.size();i++){
        a*=num_list[i];
        b+=num_list[i];
    }
    if (a<b*b){
        return 1;
    }
    return 0;
}